crates/cxx-qt-lib-extras-headers/include/gui/qapplication.h

```diff
@@ -0,0 +1,29 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Laurent Montel <laurent.montel@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+#pragma once
+
+#include <memory>
+
+#include <QtGui/QFont>
+#include <QtWidgets/QApplication>
```

Comment:
wonder if you are simply missing the QtWidgets module when building ?

In the `cxx-qt-lib-extras-headers/lib.rs` can you try changing the last `opts` to the following so that the Gui and Widgets module are imported ?  ...

```rust
opts.qt_module("Gui").qt_module("Widgets")
```

Reply:
Seems to have worked :-)